sources/ClangSharp.PInvokeGenerator/PInvokeGeneratorConfiguration.cs

```diff
@@ -267,6 +267,8 @@ public IReadOnlyCollection<string> ExcludedNames
 
     public bool StripEnumMemberTypeName => _options.HasFlag(PInvokeGeneratorConfigurationOptions.StripEnumMemberTypeName);
 
+    public bool DontUseUsingStaticsForGuidMember => _options.HasFlag(PInvokeGeneratorConfigurationOptions.DontUseUsingStaticsForGuidMember);
```

Comment:
There is some corresponding code in Program.cs of ClangSharpPInvokeGenerator that needs updating to actually set this flag (Silk.NET's ResponseFileHandler is a manually maintained subset of this Program.cs file)

Reply:
good catch, fixed it